coruscant/src/containers/App.js

```diff
@@ -17,6 +18,8 @@ const mapDispatchToProps = {
 @connect(mapStateToProps, mapDispatchToProps)
 class App extends Component {
   async componentDidMount() {
+    document.body.style.backgroundColor = Theme.palette.bodyBackground
```

Comment:
Can we instead do this with a style prop rather than directly modifying the DOM?

Reply:
If you can figure out how, I would love to hear it. I wasted more than an hour trying to figure that out. This was the best answer I could find!